File: tmdb-core/src/main/java/dev/reuss/tmdb/domain/tv/episodegroup/TvEpisodeGroupPaths.java
package dev.reuss.tmdb.domain.tv.episodegroup;

import dev.reuss.tmdb.value.id.TvEpisodeGroupId;

final class TvEpisodeGroupPaths {

    private static final String TV_EPISODE_GROUP = "/tv/episode_group";

    private TvEpisodeGroupPaths() {
    }

    static String details(TvEpisodeGroupId episodeGroupId) {
        return TV_EPISODE_GROUP + "/" + episodeGroupId.asString();
    }
}

File: tmdb-core/src/main/kotlin/dev/reuss/tmdb/domain/tv/episodegroup/TvEpisodeGroupPaths.kt
package dev.reuss.tmdb.domain.tv.episodegroup

import dev.reuss.tmdb.core.path.tmdbPath
import dev.reuss.tmdb.value.id.TvEpisodeGroupId

internal object TvEpisodeGroupPaths {
    fun details(episodeGroupId: TvEpisodeGroupId): String = tmdbPath("tv", "episode_group", episodeGroupId)
}
